### DISCUSSION TITLE
How would I get rid of the advancement text that shows when sending a notification?

### DISCUSSION THREAD

**Discussion by jordqubbe** [Minestom/Minestom#2376] [Help]
![image](https://github.com/user-attachments/assets/ae7d99bf-602d-4971-8ec5-56f4fcd1060e)

I've been figuring this out, and I can't seem to find a way to get rid of or change the text that says Challenge Complete! (or the equivalent when you change the FrameType) when you send a notification to a player. It's just basic copy-and-paste code from the Minestom wiki page right now, but I've searched through the JavaDocs and the decompiled class code to try to find a way to get rid of it, but can't seem to. Any help would be appreciated.

Here's my code (with all the imports done just not pasted):

```java
public class TestCommand extends Command {

    public TestCommand() {
        super("test");
        setDefaultExecutor((sender, context) -> {
            Notification notification = new Notification(
                    Component.text("Hello, Notifications!", NamedTextColor.GREEN),
                    FrameType.CHALLENGE,
                    ItemStack.of(Material.GOLD_INGOT)
            );
            NotificationCenter.send(notification, (Player) sender);
        });
    }

}
```

**Comment by znopp** [comment]
I looked in the javadocs and can't find anything that would allow you to set the title. It is part of the advancement toast no matter which frametype you choose. I could be wrong, but I can't find a way to remove the text.

**Comment by hapily04** [comment]
You can't right now. I looked in the 1.21.1 client code and it's hard coded to be relative to the frametype.

**Comment by Meca-3D** [comment]
I know it's been 20 days but you can remove it with texture pack (search for the a .json file for language)
